scripts/analyze_csv_data.py

```diff
@@ -0,0 +1,771 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#  This script generates all the experimental results used in the paper.
+#  It requires numpy, pandas, and matplotlib installed to run.
+#
+#   `python analyze_csv_data.py`
+#
+
+
+import os
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import matplotlib.ticker as ticker
+from cycler import cycler
+plt.rcParams.update({'figure.max_open_warning': 0, 'font.size': 16})
+
+# This is required to generate plots that are easy to read when printed:
+plt.rcParams['axes.prop_cycle'] = cycler(
```

Comment:
Note: we are mostly following Google style with this project.  So the indent levels, among other things, are wrong.  But I don't want to delay this PR with style nits, so created Issue #81 to track linter issues.